public/locales/sp/translation.json

```diff
@@ -879,13 +879,14 @@
     "sortingCode": "Código de Ordenamiento",
     "state": "Estado / Provincia",
     "displayImage": "Mostrar imagen",
-    "userRegistrationRequired": "Registro de usuario requerido",
+    "Is Public": "Es público",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.